### PR TITLE
fix style and docs

### PR DESCRIPTION
**Pull request by Cor3Down**
such as:
fix missing blank lines
fix typo mistakes
update docstrings
implement abstract methods
remove unused imports
...
so we can work better, together!

### CODE REVIEW

**Comment by Cor3Down** [comment]
hi @coveralls . I am new in contributing to open source projects and this is my first pull request.please help me to pass coverage/coveralls check. Thanks

**Comment by Cor3Down** [comment]
Hi @JeanKossaifi . I love your project and i wanna contribute to it but i am new in contribute to open source projects. please help me to work together. Thanks!

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+sparse
+tensorflow
+torch
+mxnet
```
Comment (JeanKossaifi) [inline requirements.txt]: I didn't include these on purpose: they are optional.
We want to maintain minimal dependencies.

File: tensorly/backend/__init__.py [hunk 2]
```diff
@@ -178,6 +186,8 @@ def inner(*args, **kwargs):
     return inner
 
 # Generic methods, exposed as part of the public API
+
+
```
Comment (JeanKossaifi) [inline tensorly/backend/__init__.py]: Why did you add these lines here after the comment?

File: tensorly/backend/core.py [hunk 3]
```diff
@@ -592,6 +591,9 @@ def kr(self, matrices, weights=None, mask=None):
                 for i in len(matrices):
                     matrices[i].shape = (n_i, m)
 
+        weights : should be complete by writing description about parameter
+        mask : should be complete by writing description about parameter
+
```
Comment (JeanKossaifi) [inline tensorly/backend/core.py]: We need a real description here not a place-holder, e.g.
```
mask : ndarray a mask to be applied to the final tensor. It should be
        broadcastable to the shape of the final tensor, that is
        ``(U[1].shape[0], ... U[-1].shape[0])``.
weights: 1d array of length the number of columns of the matrix
       optional weigths for each column (useful for Kruskal tensors)
```

File: tensorly/backend/cupy_backend.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+from abc import ABC
+
```
Comment (JeanKossaifi) [inline tensorly/backend/cupy_backend.py]: I do not think we want to inherit from ABC.

File: requirements.txt (hunk 1 above)
Comment (Cor3Down) [reply requirements.txt]: yes, that's right.

File: tensorly/backend/__init__.py (hunk 2 above)
Comment (Cor3Down) [reply tensorly/backend/__init__.py]: these lines created unintentionally.

File: tensorly/backend/core.py (hunk 3 above)
Comment (Cor3Down) [reply tensorly/backend/core.py]: yeah , you are right. that is my fault.i told you i am new in contributing to open source projects. i will open an issue hereinafter rather than complete its description with unusable content.

Comment (JeanKossaifi) [reply tensorly/backend/core.py]: >  i am new in contributing to open source projects

That's why I left comments with appropriate suggestions :)

> i will open an  issue hereinafter

No need, simply add a commit to this pull request with the changes!

**Comment by Cor3Down** [comment]
hi @JeanKossaifi . what is coverage/coveralls?!
and how can my pull request pass this check?

**Comment by JeanKossaifi** [comment]
Don't worry about coveralls. Since you are not touching the code, simply the doc, it is irrelevant here, just push the changes discussed in the comments.

**Comment by JeanKossaifi** [comment]
Why did you end up closing this PR?
